### DISCUSSION TITLE
how to set delay  time in a dry pipe system is not working in PyroSim

### DISCUSSION THREAD

**Discussion by ChickenBear97** [firemodels/fds#13266] [Q&A]
![1](https://github.com/user-attachments/assets/874b867a-624b-4c50-8658-2c9d623dc8df)
![2](https://github.com/user-attachments/assets/e318a168-d411-41ee-a3cf-6fd701cafeb0)
![3](https://github.com/user-attachments/assets/58cfc0a8-dab7-48f3-8843-17798d526292)
![4](https://github.com/user-attachments/assets/dccdcb5d-deff-4395-8721-6b15532efa17)
I set the delay as shown in the picture, but the sprinkler works as soon as it reaches the set temperature.
Please tell me the solution

**Comment by drjfloyd** [comment]
This forum does not provide Pyrosim support; only for FDS.  We can look at the FDS file that Pyrosim creates and see if there is a possible Pyrosim big, but for support using Pyrosim contact Thunderhead Engineering
